src/wmtk/SimplicialComplex.cpp

```diff
@@ -209,7 +210,7 @@ bool SimplicialComplex::edge_collapse_possible_2d(const TriMesh& m, const Tuple&
         return false;
     }
 
-    auto opp = [&m](const Tuple& t) { return m.switch_vertex(m.switch_face(t)); };
+    auto opp = [&m](const Tuple& tlambda) { return m.switch_vertex(m.switch_face(tlambda)); };
```

Comment:
I don't see any advantage of the renaming here but I also don't really care, so you can leave it if you want.